src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -502,6 +502,12 @@ bool DirectoryService::ProcessDSBlockConsensus(
     lock_guard<mutex> g(m_mutexConsensus);
 
     bool result = m_consensusObject->ProcessMessage(message, offset, from);
+
+    if (!result)
+    {
+        return result;
+    }
```

Comment:
Since when ProcessMessage return false we will directly return it, we don't need an extra variable to remember the result.
Suggest to rewrite as following:

if(!m_consensusObject->ProcessMessage(message, offset, from))
{
    return false;
}

And in the end of this function (line 536), we can always return true